docs/tools/vdb_table/data/nucliadb.json

```diff
@@ -26,127 +26,127 @@
     "source_url": "",
     "comment": ""
   },
-  "github_stars": 497,
-  "vector_launch_year": 0,
+  "github_stars": 537,
+  "vector_launch_year": 2021,
   "metadata_filter": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "facets": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "geo_search": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "multi_vec": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/README.md",
     "comment": ""
   },
   "sparse_vectors": {
-    "support": "",
+    "support": "none",
     "source_url": "",
     "comment": ""
   },
   "bm25": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/docs/internal/SEARCH.md",
     "comment": ""
   },
   "full_text": {
-    "support": "",
-    "source_url": "",
+    "support": "full",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/nucliadb_node/README.md#nucliadb_paragraphs-and-nucliadb_fields",
     "comment": ""
   },
   "embeddings_text": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://github.com/nuclia/nucliadb/blob/main/nucliadb_node/README.md#nucliadb_paragraphs-and-nucliadb_fields",
+    "comment": "Done in the cloud"
   },
   "embeddings_image": {
-    "support": "",
+    "support": "partial",
     "source_url": "",
-    "comment": ""
+    "comment": "Done in the cloud"
   },
   "embeddings_structured": {
-    "support": "",
-    "source_url": "",
-    "comment": ""
+    "support": "partial",
+    "source_url": "https://docs.nuclia.dev/docs/guides/using/ner/",
```

Comment:
This column is for the ability to compute graph embeddings. I don't think the NER feature linked is the same. Please let me know if I'm missing something